Nightly failure: sanity/restaked.py

Got broken between 38d54b and 2cef41.

Fails consistently since then. Example runs:

http://nightly.neartest.com/run/2cef4105736ab969470df2751e78fd3f0fc0a81d_200428_102644
http://nightly.neartest.com/run/f625606891dec008a572ca61ec37da500c2a95cd_200428_135923

Failure: 

```
Traceback (most recent call last):
  File "tests/sanity/restaked.py", line 61, in <module>
    assert False, "Validator test0 is not in current validators after 5 epochs"
AssertionError: Validator test0 is not in current validators after 5 epochs
```

--- a/pytest/tests/sanity/restaked.py
+++ b/pytest/tests/sanity/restaked.py
@@ -49,7 +49,8 @@ nodes[0].kill()
 while time.time() - started < TIMEOUT:
     status = nodes[1].get_status()
     height = status['sync_info']['latest_block_height']
-    if height > EPOCH_LENGTH * 5:
+    # epoch boundary may have shifted due to validator being offline
+    if height > EPOCH_LENGTH * 5 + 5:
         # 5 epochs later.
         validators = nodes[1].get_validators()['result']
         present = False

